pkg/catalog/types.go

```diff
@@ -72,6 +72,9 @@ type MeshCataloger interface {
 	// ListAllowedOutboundServiceAccounts lists the upstream service accounts the given service account can connect to
 	ListAllowedOutboundServiceAccounts(service.K8sServiceAccount) ([]service.K8sServiceAccount, error)
 
+	// ListMeshServices returns all services in the mesh
+	ListMeshServices() []service.MeshService
```

Comment:
What's the reason to expose this method in the interface? If it is internal to the `MeshCatalog`, it doesn't need to be exposed so wondering the use case.

Reply:
That is a very good point and I don't have a reason for exposing this at this time, so I will keep this private unless @snehachhabria objects.